src/lib/settings/schema/SchemaEntry.js

```diff
@@ -9,7 +13,7 @@ class SchemaEntry {
 	 * @typedef {Object} SchemaEntryOptions
 	 * @property {*} [default] The default value for the key
 	 * @property {Function} [filter] The filter to use when resolving this key. The function is passed the resolved value from the resolver, and a guild.
-	 * @property {boolean} [array] Whether the key should be stored as Array or not
+	 * @property {?GroupBase} [group] The group that holds this instance
```

Comment:
For this, and seemingly everything referring to it, we need to make a clear and defined way of setting things, rather then saying if A isn't valid, use or B, else if B isn't valid, use A. Make this more streamlined and easier to use